runtime/lib/app/activity-descriptor.js

```diff
@@ -228,6 +228,15 @@ Object.assign(ActivityDescriptor.prototype,
      */
     notification: {
       type: 'event'
+    },
+    /**
+     * Fires on app event.
+     * @event yodaRT.activity.Activity#appcmd
+     * @param {string} intent - the intent of app event.
+     * @param {object} data - the extra data.
+     */
+    appcmd: {
```

Comment:
In which conditions this event would be triggered? Would prefer `notification` instead of other new event.